RELEASE_NOTES.md

```diff
@@ -2,6 +2,7 @@
 
 - fix: Compiled lambdas now close only on non-ghost variables (https://github.com/dafny-lang/dafny/pull/2854)
 - fix: Crash in the LSP in some code that does not parse (https://github.com/dafny-lang/dafny/pull/2833)
+- fix: Exclude ghost variables from compilation of lambda-expression closures (https://github.com/dafny-lang/dafny/pull/2873)
```

Comment:
I think your fix is the same as the fix of the PR I filed 5 days ago 😁
https://github.com/dafny-lang/dafny/pull/2854

Reply:
Oh, by golly, you're right.